SI/Sem.04/Pract.04/README.md

```diff
@@ -4,51 +4,56 @@
 
 2. Да се напише функция isEven(int num), която връща дали дадено число е четно или не.
 
-3. Да се напише функция isPositive(int num), която връща дали дадено число е положително или не
+3. Да се напише функция 
+   - isPositive(int num), която връща дали дадено число е положително или не
+   - isNegative(int num), която връща дали дадено число е отрицателно или не.
 
-4. Да се напише функция isNegative(int num), която връща дали дадено число е отрицателно или не.
-
-5. Напишете функция abs(int num), която връща като резултат абсолютната стойност на числото num.
-
-6. Напишете функция fabs(double num), която връща като резултат абсолютната стойност на числото num.<br />
+4. Напишете функция 
+   - abs(int num), която връща като резултат абсолютната стойност на числото num.
+   - fabs(double num), която връща като резултат абсолютната стойност на числото num.<br />
     *Hint: Epsilon*
 
-7. Напишете функция print(char S, int N), която извежда на екрана N-пъти символа S.
+5. Напишете функция print(char S, int N), която извежда на екрана N-пъти символа S.
 
-8. Напишете функция isCapitalLetter(char S), която проверява дали даден символ е главна буква.
-   *Реализирайте функция checkSymbol(char S), която да проверява дали символът е буква.*
+6. Напишете функция isCapitalLetter(char S), която проверява дали даден символ е главна буква. <br />
+  *Реализирайте функция checkSymbol(char S), която да проверява дали символът е буква.* <br />
 
-9. Напишете функция, която проверява дали дадено число е в даден интервал.
- 
-10. Напишете функция pow(int num, unsigned N), която връща като резултат числото num повдигнато на степен N, където N е строго положително.
+7. Напишете функция
+   - която проверява дали дадено число е в даден интервал. 
+   - askUser(int from, int to), която кара потребителя да въвежда числа докато не въведе число,
+     което да е в указания интервал, а като резултат връща въпросното число.
+  
+8. Напишете функция pow(int num, unsigned N), която връща като резултат числото num повдигнато на степен N, където N е строго положително.
 
-11. Напишете функция calculate(int a, int b, char action), където action може да бъде [+-*/%].
+9. Напишете функция calculate(int a, int b, char action), където action може да бъде [+-*/%].
   Програмата да връща като резултат съответното действие, а ако не се укаже символ да върне сбора на числата.
 
-12. Напишете функция, askUser(int from, int to), която кара потребителя да въвежда числа докато не въведе число,
-  което да е в указания интервал, а като резултат връща въпросното число.
+10. Напишете програма, която по въведено число изкарва представянето му в двоична бройна система. <br />
```

Comment:
They will learn binary on week 8

Reply:
Bitwise operators are week 8.
Binary should be known or if not , we will address it.